spec/visual/samples/eex

```diff
@@ -0,0 +1,45 @@
+<h3>List</h3>
+<%= if Enum.empty?(@list) do %>
+<p>not found.</p>
+<% else %>
+<table>
+  <tbody>
+    <%= for {item, i} <- Enum.with_index(list) do %>
+    <tr bgcolor="<%= if Integer.mod(i, 2) == 0, do: "#FFCCCC", else: "#CCCCFF" %>">
```

Comment:
This line fails to lex properly because of a limitation in the Elixir lexer. Specifically here:

https://github.com/rouge-ruby/rouge/blob/59624c143eccd72b5d9a823888de3cfcea3f175d/lib/rouge/lexers/elixir.rb#L80-83

It looks to me like the problem is that we need a rule to match `#` in a string that aren't interpolations. I'm not very familiar with Elixir but I gather from the lexer that an interpolation is of the form `#{...}`. Is that right, @julp?

Reply:
Yes, you're right, same as Ruby.

Reply:
OK. Will submit a fix for the Elixir lexer.

Reply:
Fix submitted as #1308.